src/desktop/components/auth_modal/view.coffee

```diff
@@ -70,6 +70,9 @@ module.exports = class AuthModalView extends ModalView
 
     @logState()
 
+    Cookies.set 'postSignupAction', JSON.stringify(@afterSignUpAction) if @afterSignUpAction
+    Cookies.set('destination', @destination, expires: 60 * 60 * 24) if @destination
```

Comment:
Missed this the other day! When doing sign up via Facebook (which you have to click around in the modal to even access, the 'Register' modal flow emphasizes email/password), this never gets set. Now you'll get the correct post-onboarding behavior when signing up via facebook as well.